interfaces/final-object.interface.ts

```diff
@@ -39,6 +39,7 @@ export interface ImageElement {
   stars: StarRating;             // star rating 0 = n/a, otherwise 1, 2, 3
   timesPlayed: number;           // number of times the file has been launched by VHA
   width: number;                 // width of the video (px)
+  frameRate: number;            //  base frame rate of the video in fps
```

Comment:
I don't have a strong preference, but perhaps `fps` is better - as it's shorter and is already a well-known acronym 🤔